drag_and_drop_v2/public/css/drag_and_drop_edit.css

```diff
@@ -222,6 +242,7 @@
     display: none;
 }
 .xblock--drag-and-drop--editor .items-form .row.advanced-link {
+    cursor: pointer;
```

Comment:
Instead of using an a-tag, you should use a button. A-tag elements should take you someplace, whereas buttons do actions. Buttons give you the correct a11y behavior, whereas you have to manually create those behaviors with other elements using aria properties.

Tabbing through the Studio editing UI, I don't see any way to even get to these "Remove" links with the keyboard.